draft-dr-dprive-tls-and-dtls-authentication.txt

```diff
@@ -72,62 +79,63 @@ Table of Contents
    11. Server capability probing and caching by DNS clients  . . . .  10
    12. (D)TLS Protocol Profile . . . . . . . . . . . . . . . . . . .  10
-   13. IANA Considerations . . . . . . . . . . . . . . . . . . . . .  11
-   14. Security Considerations . . . . . . . . . . . . . . . . . . .  11
-   15. Acknowledgements  . . . . . . . . . . . . . . . . . . . . . .  11
-   16. References  . . . . . . . . . . . . . . . . . . . . . . . . .  11
-     16.1.  Normative References . . . . . . . . . . . . . . . . . .  11
-     16.2.  Informative References . . . . . . . . . . . . . . . . .  12
-   Appendix A.  Changes between revisions  . . . . . . . . . . . . .  13
-   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  13
+   13. IANA Considerations . . . . . . . . . . . . . . . . . . . . .  12
+   14. Security Considerations . . . . . . . . . . . . . . . . . . .  12
+   15. Acknowledgements  . . . . . . . . . . . . . . . . . . . . . .  12
+   16. References  . . . . . . . . . . . . . . . . . . . . . . . . .  12
+     16.1.  Normative References . . . . . . . . . . . . . . . . . .  12
+     16.2.  Informative References . . . . . . . . . . . . . . . . .  13
+   Appendix A.  Changes between revisions  . . . . . . . . . . . . .  14
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  14
 
 1.  Introduction
 
-   To address the concerns in [RFC7626] the DPRIVE working group has two
-   active documents that provide DNS privacy between DNS stubs and
-   recursive resolvers:
-
-   o  DNS-over-TLS [I-D.ietf-dprive-dns-over-tls]
-
-   o  DNS-over-DTLS [I-D.ietf-dprive-dnsodtls]
 
 
 
 
 
-Dickinson & Reddy         Expires May 19, 2016                  [Page 2]
+Dickinson & Reddy         Expires May 21, 2016                  [Page 2]
 
 Internet-Draft            (D)TLS Authentication            November 2015
 
 
+   The DPRIVE working group has two active documents that provide DNS
+   privacy between DNS clients and DNS servers (to address the concerns
+   in [RFC7626]):
+
+   o  DNS-over-TLS [I-D.ietf-dprive-dns-over-tls]
+
+   o  DNS-over-DTLS [I-D.ietf-dprive-dnsodtls]
+
    This document defines authentication profiles for DTLS [RFC6347] and
-   TLS [RFC5246] that specify how a DNS stub client or forwarder should
-   authenticate the DNS recursive resolver it talks to, based on a
-   domain name associated with that resolver.  In particular, it
-   describes:
+   TLS [RFC5246] that specify how a DNS client should authenticate a DNS
+   server based on a domain name.  In particular, it describes:
+
+   o  How a DNS client can obtain a domain name for a DNS server.
 
-   o  How a DNS client can obtain a domain name for the DNS server it
-      talks to.
+   o  What are the acceptable credentials a DNS server can present to
+      prove its identity in relation to a given domain name.
 
-   o  What sorts of credentials a DNS server can present to prove its
-      control over a given domain name.
```

Comment:
"Control over a given domain name" could be construed in a DNS sense to mean server the zone for that domain name